db/migrate/20150520174217_add_index_to_decks.rb

```diff
@@ -0,0 +1,8 @@
+class AddIndexToDecks < ActiveRecord::Migration
+  def change
+    change_table :decks do |t|
```

Comment:
Unused block argument - `t`. You can omit the argument if you don't care about it.